pkgs/os-specific/linux/molly-guard/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchurl, dpkg, busybox, systemd }:
+
+stdenv.mkDerivation rec {
+  name = "molly-guard";
```

Comment:
Add version to name. Like:
```
name = "molly-guard-${version}";
```

`version` field has no meaning for `mkDerivation`.